CONTRIBUTING.md

```diff
@@ -70,22 +70,30 @@ helper methods.
 
 #### How to enable and configure
 
-* Create a folder in your project called `.publicApi` with the frameworks that
+1. Create a folder in your project called `.publicApi` with the frameworks that
```

Comment:
What needs to be done here is a bit more nuanced 🤣 

Here are the rules:

* If you are NOT using experimental APIS...

  * If your API is the same for all targets...

    You only need two files: `.publicApi/PublicAPI.Shipped.txt` and `.publicApi/PublicAPI.Unshipped.txt`

  * If your APIs differ between targets...

    a) Put all the shared APIs into these two files: `.publicApi/PublicAPI.Shipped.txt` and `.publicApi/PublicAPI.Unshipped.txt`
  
    b) Create framework-specific files for the API differences: `.publicApi/net462/PublicAPI.Shipped.txt` and `.publicApi/net462/PublicAPI.Unshipped.txt`

    Example: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Api/.publicApi/Stable

* If you are using experimental APIs...

  * Follow the rules as above except create an additional layer in your folder structure: `.publicApi/Experimental/*` and `.publicApi/Stable/*`
  
    The `Experimental` folder should contain APIs which are `public` only in pre-release builds.

    Example: `https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry/.publicApi`